treeherder/perf/auto_perf_sheriffing/perf_sheriff_bot.py

```diff
@@ -204,3 +217,81 @@ def _notify_backfill_outcome(self):
 
         # send email
         self._notify.email(backfill_notification)
+
+    def _handle_backfill_alerts(self):
+        # Get backfilled alerts
+        bugzilla = BugzillaHelper()
+        backfilled_records = BackfillRecord.objects.select_related(
+            'alert', 'alert__series_signature', 'alert__series_signature__platform'
+        ).filter(
+            # TODO: Enable this filter once we start getting more backfills
+            # status=BackfillRecord.SUCCESSFUL,
+            alert__series_signature__platform__platform__icontains='linux',
+        )
+
+        for record in backfilled_records:
+            # Start by recomputing the alert to see if we have a new
+            # culprit commit
+            # TODO: Use a try/catch on this later rather than the changed variable
+            try:
+                cur_push, prev_push, changed = AlertManager.recompute_backfill_alert(record)
+                if changed is None:
+                    logger.warning(f"Failed to recompute the alert in record {record}")
+                    continue
+            except AlertInvalidated as e:
+                # TODO: Mark the alert as invalid
+                logger.info(
+                    f"The alert that was backfilled is now invalid. Reason: {e}"
+                )
+                continue
+
+            # Check if this push has an alert summary already. If it does, reassign
+            # the alerts to it, delete this summary.
+            existing_summary = PerformanceAlertSummary.objects.filter(push=cur_push)
```

Comment:
`PerformanceAlertSummary.objects.filter(push=cur_push).first()` gets the first one or `None` if there is no result after filtering